### DISCUSSION TITLE
how to set_value typle is "word" value????

### DISCUSSION THREAD

**Discussion by westbullman** [FreeOpcUa/python-opcua#1482] [Q&A] [answered]
my code is 
 " dv = ua.DataValue(ua.Variant(1, ua.VariantType.Word))
  thisResultItem.set_value(dv)"

this doesn't work，who can help me？thanks !!

**Comment by swamper123** [comment, marked as answer]
Hey ho,

first things first - please read #1475 

To your problem:
`Word` isn't an inbuilt type of OPC UA.
Here is the source of the VarianTypes from this (deprecated) library:
https://github.com/FreeOpcUa/python-opcua/blob/96a8ea672d09ab8bc78388a403998b0baa14fa95/opcua/ua/uatypes.py#L624

You need your nodeset file (I guess from SIOME or PLCOpen?), import it, load the datatype definition and then you can create and interact with a node.

Importing a Nodeset:
https://github.com/FreeOpcUa/python-opcua/blob/master/examples/server-import-robot-nodeset.py

Creating a node to use (from a nodeset):
https://github.com/FreeOpcUa/python-opcua/blob/858274721766907148b527916e97446dbbf6af6d/examples/server-create-custom-structures.py
